best-practices-trove-project-creation.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "Creating projects for Trove"
+permalink: /best-practices-trove-project-creation/
+---
+# Creating a data collection project in Trove
+
+With Trove, you can create AI/ML data collection projects through trusted connections between you and the people who contribute to your projects, leading to higher quality data. This document explains how to create such a project in Trove.
+
+## Create Project in Trove Studio
+
+Start your data collection project by signing into the [Trove Studio](https://trove-studio.microsoft.com/) as a requester.  Once your account has been verified, click the "Create project" link to provide the details of your project.
```

Comment:
I am not a big fan of the term "requester". if it's a market, i assume we would say "buyer"?

Reply:
Great point as I used requester directly from the home page of Trove Studio which ask visitors to sign as requester to be consistent with the flow of the process.  I also brought this up in daily scrum stand-up and understood from a language assessment a couple of years, there are internal and external terminologies for requester (buyer) and provider (seller).  I will create a bug to change the text of the Trove Studio sign-in button to show "Sign in as buyer" instead of requester.
![TroveStudioSignIn](https://user-images.githubusercontent.com/19422202/129596444-907a73c7-9816-4f59-83af-d585f6be5b92.png)

Reply:
Set this to resolved for now as I have logged a bug to change the label for the Sign-in button to say buyer - https://dev.azure.com/msctoproj/DataDignity/_workitems/edit/14811/